### PR TITLE
Super shardian page with airtable data without localisation

### PR DESCRIPTION
**Pull request by codeReadyy**

### CODE REVIEW

File: pages/supershardians/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            : { selected: false, name: category.name }
+        )
+      );
+    }
+  };
+  return (
+    <>
+      <Hero
+        heading={"Our Super Shardians"}
+        description={
```
Comment (dewanshparashar) [inline pages/supershardians/index.tsx:51]: Can we make it translation ready?
Going forward we had decided that.

File: utils/api.ts [hunk 2]
```diff
@@ -55,3 +59,27 @@ export const getSHMNewsArticles = (): Promise<NewsItem[]> => {
...
+export function getSuperShardians() {
+  configureAirtable();
+  const data: any[] = [];
+  const base = Airtable.base(process.env.NEXT_PUBLIC_AIRTABLE_BASE_ID as string);
+  return new Promise<any>((resolve, reject) => {
+    base(process.env.NEXT_PUBLIC_AIRTABLE_SUPERSHARDEUM as string)
+      .select({
+        view: "Grid view",
+      })
+      .firstPage()
```
Comment (dewanshparashar) [inline utils/api.ts:72]: Please add conditions for null check in this chain

File: components/sections/supershardians/ShardiansList.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { Button, Container, Flex, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
+
+type ShardiansProps = {
+  categories: Array<any>;
+  filteredData: Array<any>;
+  changeCategory: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+const Shardians = ({ categories, filteredData, changeCategory }: ShardiansProps) => {
```
Comment (dewanshparashar) [inline components/sections/supershardians/ShardiansList.tsx:10]: Breaks at this resolution
image

File: components/sections/supershardians/ShardiansList.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <Flex bg="brand.grey-5">
+      <Container pl={0} pr={0} maxW="container.xl" pt={[6, 8, 12]} pb={[6, 8, 12]}>
+        <HStack pb={12} pl={[3, 5, 0]} pr={[3, 5, 0]}>
+          {categories.map((category, index) => {
+            return (
+              <Button
```
Comment (dewanshparashar) [inline components/sections/supershardians/ShardiansList.tsx:17]: On click of the filter button - this focus state gets activated which looks wierd.  
image

File: pages/supershardians/index.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
+import { Box, Button, Container, filter, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
```
Comment (dewanshparashar) [inline pages/supershardians/index.tsx:2]: Can we remove the components which are imported but not being used?

image

File: components/sections/supershardians/BeASuperShardians.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import { Box, Button, Container, Flex, SimpleGrid, Text, VStack } from "@chakra-ui/react";
```
Comment (dewanshparashar) [inline components/sections/supershardians/BeASuperShardians.tsx:1]: Remove the components not being used.

image

File: components/sections/supershardians/BeASuperShardians.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+          </Box>
+          <Button
+            size="lg"
+            as="a"
+            bg="brand.grey-5"
+            color="brand.grey-90"
+            target="_blank"
+            href={COMMUNITY_URL}
+          >
+            Start Here
```
Comment (dewanshparashar) [inline components/sections/supershardians/BeASuperShardians.tsx:38]: Import it from translations json file

File: pages/supershardians/index.tsx [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+        categories.map((category) =>
+          event.target?.value === category.name
+            ? { selected: true, name: category.name }
+            : { selected: false, name: category.name }
+        )
+      );
+    }
+  };
+  return (
+    <>
```
Comment (dewanshparashar) [inline pages/supershardians/index.tsx:48]: We should also have SEO tags <meta > for this page like in other pages. Pls contact PM's and add the tags here..

File: components/common/ProfileInfoCard.tsx [hunk 9]
```diff
@@ -0,0 +1,46 @@
+import {
+  Flex,
+  Circle,
```
Comment (dewanshparashar) [inline components/common/ProfileInfoCard.tsx:3]: Currently this page is not accessible from anywhere.
Pls add a link in navbar or any other place where user will click to reach Super-Shardians page..

File: utils/api.ts [hunk 10]
```diff
@@ -55,3 +59,27 @@ export const getSHMNewsArticles = (): Promise<NewsItem[]> => {
...
+      .then((records) => {
+        records.forEach(function (record) {
+          const name = record.get("Name");
+          const description = record.get("Description");
+          const createdDate = record.get("CreatedDate");
```
Comment (dewanshparashar) [inline utils/api.ts:77]: Variable assigned but never used

**Review by dewanshparashar — CHANGES_REQUESTED** [review]
Some changes pending before we merge and release to production..

File: pages/supershardians/index.tsx (hunk 5 above)
Comment (codeReadyy) [reply pages/supershardians/index.tsx:2]: Removed

File: components/sections/supershardians/ShardiansList.tsx (hunk 4 above)
Comment (codeReadyy) [reply components/sections/supershardians/ShardiansList.tsx:17]: Addressed

File: components/sections/supershardians/ShardiansList.tsx (hunk 3 above)
Comment (codeReadyy) [reply components/sections/supershardians/ShardiansList.tsx:10]: Fixed

File: components/common/ProfileInfoCard.tsx (hunk 9 above)
Comment (codeReadyy) [reply components/common/ProfileInfoCard.tsx:3]: I will connect with product team for the same as it is not in design, for now adding it to navbar.

File: pages/supershardians/index.tsx (hunk 1 above)
Comment (codeReadyy) [reply pages/supershardians/index.tsx:48]: I'll create a separate PR for this, for now let's go live with this.

File: utils/api.ts (hunk 2 above)
Comment (codeReadyy) [reply utils/api.ts]: Done

File: components/sections/supershardians/BeASuperShardians.tsx (hunk 7 above)
Comment (codeReadyy) [reply components/sections/supershardians/BeASuperShardians.tsx:39]: Same as above

File: pages/supershardians/index.tsx (hunk 8 above)
Comment (codeReadyy) [reply pages/supershardians/index.tsx:45]: PMs would take more time to provide these tag, Point noted I will have these changes along with translation PR.

File: utils/api.ts (hunk 10 above)
Comment (codeReadyy) [reply utils/api.ts:77]: Removed

File: pages/supershardians/index.tsx (hunk 1 above)
Comment (dewanshparashar) [reply pages/supershardians/index.tsx]: Have added translation

File: components/sections/supershardians/BeASuperShardians.tsx (hunk 7 above)
Comment (dewanshparashar) [reply components/sections/supershardians/BeASuperShardians.tsx]: Added translation

File: components/sections/Navbar.tsx [hunk 11]
```diff
@@ -15,6 +15,11 @@ const links = [
     link: COMMUNITY_URL,
     newPage: true,
   },
+  {
```
Comment (dewanshparashar) [inline components/sections/Navbar.tsx:18]: This page is highlighting the navbar issue. That will be needed to be pushed separately before going live.

image

Comment (codeReadyy) [reply components/sections/Navbar.tsx:19]: Hero component bg color changed to black

File: pages/supershardians/index.tsx (hunk 8 above)
Comment (codeReadyy) [reply pages/supershardians/index.tsx]: This is done

File: components/sections/supershardians/ShardiansList.tsx [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+              >
+                {category.name}
+              </Button>
+            );
+          })}
+        </HStack>
+        <SimpleGrid spacing={["20px", "30px"]} columns={[1, 2, 3]}>
+          {filteredData.map((data, index) => {
+            return (
+              <Card
```
Comment (dewanshparashar) [inline components/sections/supershardians/ShardiansList.tsx:36]: Overflow for text is still not handled
image

File: pages/super-shardian/index.tsx [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+        )
+      );
+    }
+  };
+  return (
+    <>
+      <NextSeo
+        title={ssTranslation("page-super-shardian-seo-title")}
+        description={ssTranslation("page-super-shardian-seo-description")}
+        canonical="https://shardeum.org/super-shardian/"
```
Comment (dewanshparashar) [inline pages/super-shardian/index.tsx:53]: SEO Tags are not to be translated. They should be static

File: components/common/ProfileInfoCard.tsx [hunk 14]
```diff
@@ -0,0 +1,34 @@
+import { Flex, Box, Image, Badge, Text, AspectRatio } from "@chakra-ui/react";
+
+type CardProps = {
```
Comment (dewanshparashar) [inline components/common/ProfileInfoCard.tsx:3]: Pls fix the lint warnings coming in console on yarn-build

image

**Review by dewanshparashar — CHANGES_REQUESTED** [review]
More review comments identified.

**Review by dewanshparashar — APPROVED** [review]
LGTM

**Pull request merged**